libs/store-harvester/mocked/state.ts

```
@@ -0,0 +1,20 @@
+import type { StoredData } from '../src/state/types';
+import { StateDatabase as SrcStateDatabase } from '../src/state';
+import PouchDB from 'pouchdb';
+import memory from 'pouchdb-adapter-memory';
+
+// Note: multiple imports of this are not an issue
+PouchDB.plugin(memory);
+
+declare global {
+  var __temp_state: PouchDB.Database<StoredData>;
```

ESLint

[error] All 'var' declarations must be at the top of the function scope.

(vars-on-top)